Scan/utkscan/CMakeLists.txt

```diff
@@ -34,16 +30,65 @@ if(UTKSCAN_VERBOSE)
     add_definitions(-D VERBOSE)
 endif(UTKSCAN_VERBOSE)
 
-#------------------------------------------------------------------------------
-
 #Check if GSL is installed
 if(USE_GSL)
     find_package(GSL REQUIRED)
     add_definitions("-D usegsl")
 endif(USE_GSL)
 
+#------------------------------------------------------------------------------
+
+#Add the includes from ScanLib to our tree
+include_directories(include ../ScanLib/include)
```

Comment:
This line shouldn't be here. the ScanLib/include is added in the parent CMakeLists.txt